applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -380,79 +511,62 @@ static void fix_rep_wk(struct k_work *work)
 	}
 
 #if defined(CONFIG_SLM_NRF_CLOUD)
+
```

Comment:
Additional line change.

Reply:
I don't think that this is official notation for our code. I am not against having this or others like this, but I do not want to force people to follow this approach.

Reply:
Can be resolved.